### ISSUE TITLE
connection is open to MITM attack

### ISSUE THREAD

**Issue by lhost** [tomaj/php-payments#8]
bug introduced in commit 8c5021053f65a272dda28d7123598361968b8f00

**Comment by tomaj** [comment]
yes this is a problem but there are compatibility issues with ubuntu openssl library with same php versions. so i decided to disable this validation.

right now there is no planing updates for this library. I am focusing more on new omnipay versions of payments here: https://github.com/tomaj/omnipay-tatra
